### PR TITLE
Bihar cleanup encore

### PR DESCRIPTION
**Pull request by orangejenny**
http://manage.dimagi.com/default.asp?170389

hobbit @proteusvacuum 
have context @snopoke / @emord

### CODE REVIEW

**Comment by dimagimon** [comment]
There are 6 errors:

* custom/bihar/management/commands/bihar_cleanup_case_2.py, line 1 - F401 'namedtuple' imported but unused
* custom/bihar/management/commands/bihar_cleanup_case_2.py, line 2 - F401 'logging' imported but unused
* custom/bihar/management/commands/bihar_cleanup_case_2.py, line 4 - F401 'ResourceNotFound' imported but unused
* custom/bihar/management/commands/bihar_cleanup_case_2.py, line 8 - F401 'iter_docs' imported but unused
* custom/bihar/management/commands/bihar_cleanup_case_2.py, line 12 - E303 too many blank lines (3)
* custom/bihar/management/commands/bihar_cleanup_case_2.py, line 72 - W391 blank line at end of file

File: custom/bihar/management/commands/bihar_cleanup_case_2.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+class Command(BaseCommand):
+    """
+    One time command for cleaning up care-bihar data.
+    Takes one argument, the directory containing the three input files.
+    """
+
+    cases_to_save = {}
+
+    def get_case(self, case_id):
+        case = self.cases_to_save.get(case_id) or get_case_by_identifier("care-bihar", case_id)
```
Comment (snopoke) [inline custom/bihar/management/commands/bihar_cleanup_case_2.py]: Are you sure you want to use `get_case_by_identifier` and not just `CommCareCase.get(case_id)`?

Comment (snopoke) [inline custom/bihar/management/commands/bihar_cleanup_case_2.py]: Checked the ticket and I'm 100% sure you should just use `CommCareCase.get` which will prevent hitting ES twice as well as couch for every case.

Comment (orangejenny) [inline custom/bihar/management/commands/bihar_cleanup_case_2.py]: fixed

**Comment by snopoke** [comment]
I'm happy to merge this but I am a bit concerned that if you end up with a lot of cases in the `cases_to_save` then the bulk save may take a very long time / not succeed. Up to you if you want to change it or just try it out first.

**Comment by orangejenny** [comment]
@snopoke Any sense of what's a good number of cases to save? I also a bit concerned about that, noticed that in the related task you did, you saved every 100 cases. This could update up to 1200 cases, depending on how many actually need to be changed.

But given that this isn't that time-sensitive, I'm inclined to try it & rework it if it fails.

**Comment by snopoke** [comment]
Yea, probably every 100 is a good bet

**Comment by czue** [comment]
1200 I think will be ok. @orangejenny are you coordinating getting this run? feel free to just kick the ticket back to interrupt of not with a pointer her.

**Comment by orangejenny** [comment]
Sent back to interrupt, updated ticket with input files & instructions.

**Pull request merged**